sw-precache-config.json

```diff
@@ -0,0 +1,14 @@
+{
+ "staticFileGlobs": [
+   "/index.html",
+   "/manifest.json",
+   "/images-optimized/**",
+   "/src/**",
+   "/bower_components/webcomponentsjs/webcomponents-lite.min.js"
+ ],
+ "navigateFallback": "/index.html",
+ "runtimeCaching": [{
+   "urlPattern": "/https:\\/\\/fonts\\.googleapis\\.com\\//",
```

Comment:
Go ahead and remove the font. lets try to go for minimal size in this repo and use system fonts only. :package: